### PR TITLE
Add migration testsuites for SLES15SP6 migration path of Texmode default pattern

### PR DESCRIPTION
**Pull request by okynos**
We have added different templates and configurations to allow the migration of sp6 jobs in OpenQA.

- Related ticket: https://progress.opensuse.org/issues/166268
- Needles: N/A
- Related MR: https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/348

### Verification runs
  - For Migrations check this job group, build 31.1: https://openqa.suse.de/tests/overview?distri=sle&version=15-SP7&build=31.1&groupid=583
  - For HDD building check build 20241107: https://openqa.suse.de/tests/overview?distri=sle&version=15-SP6&build=20241107&groupid=583

### CODE REVIEW

**Review by rakoenig — APPROVED** [review]
LGTM.

**Comment by lemon-suse** [comment]
In general, looks good. One question, have you compare the profiles with latest cloned profile to ensure important section for 15SP6 included? We need do this compare to ensure won't miss something in product (ex: 15SP6) , this may include patterns, services, and packages, etc.

File: data/yam/autoyast/support_images/autoyast_sles15sp4_sp6_media_install_textmode_default_patterns_aarch64.xml [hunk 1]
```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0"?>
```
Comment (lemon-suse) [inline data/yam/autoyast/support_images/autoyast_sles15sp4_sp6_media_install_textmode_default_patterns_aarch64.xml:1]: Please don't follow this name style except these three product profiles are completely same. Suggest to name it as autoyast_sles15sp6_media_install_textmode_default_patterns_aarch64.xml  So for other files.

**Comment by lemon-suse** [comment]
A suggestion for the VRs, you can show the group (ex: https://openqa.suse.de/tests/overview?distri=sle&version=15-SP6&build=okynos%2Fos-autoinst-distri-opensuse%23166268-migration-sp6) while not list all there. And just like review, need add comments for failures.

File: data/yam/autoyast/support_images/autoyast_sles15sp4_sp6_media_install_textmode_default_patterns_aarch64.xml (hunk 1 above)
Comment (okynos) [reply data/yam/autoyast/support_images/autoyast_sles15sp4_sp6_media_install_textmode_default_patterns_aarch64.xml]: Solved here c654766b3

**Comment by okynos** [comment]
Hello @lemon-suse I included the modifications you mentioned. Thanks!

File: data/yam/autoyast/support_images/autoyast_sles15sp6_media_install_textmode_default_patterns_s390x.xml.ep [hunk 2]
```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0"?>
```
Comment (lemon-suse) [inline data/yam/autoyast/support_images/autoyast_sles15sp6_media_install_textmode_default_patterns_s390x.xml.ep:1]: Not a dynamic profile any more, remove the extension '.ep'.

Comment (okynos) [reply data/yam/autoyast/support_images/autoyast_sles15sp6_media_install_textmode_default_patterns_s390x.xml.ep]: bc7990acb

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**